modules/touch/src/touchinteraction.cpp

```diff
@@ -759,20 +632,23 @@ int TouchInteraction::interpretInteraction(const std::vector<TouchInputHolder>&
     if (_zoomOutTap) {
         return ZOOM_OUT;
     }
-    else if (_doubleTap) {
-        return PICK;
-    }
+    //else if (_doubleTap) {
```

Comment:
I'd remove the comment alltogether.  I think when we do the picking, we'd want to handle the taps more centrally in the code anyway

Reply:
Okay, I kept it for the sake of not removing all the intention of using the `_doubleTap` variable for something else than just tapping in the bottom corner (which is currently not working anyways though). But yeah sure I'll just remove it